_tutorials/manual-build.md

```diff
@@ -10,13 +10,13 @@ redirect_from:
 
 # {{page.title}}
 
-Adding Origami components to your product through the manual build process gives you more granular control over their styling and their behaviour. It requires more set up though, so we're providing an in-depth walkthrough for building a page for an article about fruit.
+Installing Origami components with a package manager (manual build) gives you more granular control over their styling and their behaviour within your project. It requires more set up though, so we're providing an in-depth walkthrough for building a page for an article about fruit.
```

Comment:
I've left some recognition of our previous terminology for the time being, and have left the old url intact.

>with a package manager (manual build)

Reply:
Nice idea to keep manual build mentioned somewhere in the page ☺️